src/expr/src/expr/expr_literal.rs

```diff
@@ -72,29 +71,20 @@ impl<'a> TryFrom<&'a ExprNode> for LiteralExpression {
     type Error = ExprError;
 
     fn try_from(prost: &'a ExprNode) -> Result<Self> {
-        ensure!(prost.get_expr_type().unwrap() == Type::ConstantValue);
         let ret_type = DataType::from(prost.get_return_type().unwrap());
-        if prost.rex_node.is_none() {
```

Comment:
This can be removed thanks to #7936, but seems the unit test is still using the old convention (rust None for sql NULL).

Reply:
Fixed.